src/drivers/net/bnxt/bnxt.c

```diff
@@ -1823,13 +1841,13 @@ static int bnxt_hwrm_ring_alloc ( struct bnxt *bp, u8 type )
 		req->int_mode   = BNXT_CQ_INTR_MODE ( ( (FLAG_TEST ( bp->flags, BNXT_FLAG_IS_CHIP_P7) ) || bp->vf ) );
 		req->length 	= ( u32 )bp->nq.ring_cnt;
 		req->logical_id = 0xFFFF; // Required value for Thor FW?
-		req->page_tbl_addr = virt_to_bus ( bp->nq.bd_virt );
+		req->page_tbl_addr = ( unsigned long ) ( bp->nq.bd_virt );
```

Comment:
This looks implausible to me.  The old code would have filled `page_tbl_addr` with a physical (bus) address, but the replacement code looks as though it fills `page_tbl_addr` with a virtual address.

Reply:
Ack, and fixed